### DISCUSSION TITLE
Translation - Food vs Ingredient difference?

### DISCUSSION THREAD

**Discussion by auanasgheps** [TandoorRecipes/recipes#2061] [General]
Hi,

I have finally resumed work on the Italian translation, but I stumbled upon an issue: the difference between food and ingredient in this program.

In Italian:
- food is something that has been worked to be eaten, like: bread and pasta. 
- ingredient is one part of a food, like yeast, flour.

However, I don't understand the difference between food and ingredient in Tandoor.

Can somebody tell how I should handle the word "food" and how translators from other languages have dealt with it?

**Comment by vabene1111** [comment]
In tandoor food can be anything, either a raw fruit, vegetable,... Or something processed like chocolate,...ingredients are basically a tuple consisting of a food, a unit and an amount. Does that make sense?

**Reply by auanasgheps** [reply]
To be honest I don't understand it. Maybe it's just me.
I might get away with it by just calling everything "ingredients" in my own language.
